pymc3/distributions/continuous.py

```diff
@@ -4223,6 +4223,28 @@ class Interpolated(BoundedContinuous):
     Both parameters ``x_points`` and values ``pdf_points`` are not variables, but
     plain array-like objects, so they are constant and cannot be sampled.
 
+    .. plot::
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+        import pymc3 as pm
+        import arviz as az
+        from scipy.stats import gamma
+        rv = gamma(1.99)
+        x = np.linspace(rv.ppf(0.01),rv.ppf(0.99), 1000)
+        points = np.linspace(x[0], x[-1], 50)
+        pdf = rv.pdf(points)
+        interpolated = pm.Interpolated.dist(points, pdf)
+        plt.style.use('arviz-darkgrid')
```

Comment:
Minor nit, move the style call line 4233 between the imports and the code. Otherwise looks great

Reply:
done